### PR TITLE
Added French translation

### PR DESCRIPTION
**Pull request by Arsaell**
This is a first shot at a french translation for this mod.

I haven't spent a long time testing the mod in-game, so for some of the strings I had to guess in what context they would be used, and i have probably been wrong some (many ?) times. The base meaning should be there anyways, it just the word endings that might be funny.

Plus, it is my first contribution to any rimworld mod, so i haven't the slightest idea if I'm supposed to write anything elsewhere than in this languages folder ? There might be a list of supported languages somewhere, which should then be updated ?

Kuddos !

### CODE REVIEW

**Comment by jptrrs** [comment]
Génial, Remi! Merci beaucoup!
I was forced to take a break from modding last month, and only now I'm
taking some time to update my mods. But I'm thrilled to add a french
version! You're very much welcome to the team.
I'll look into your translation asap, but I'm probably going to ask you to
add some lines before I can integrate this. That's because I made some
improvements right before Tynan updated RW, and I haven't published them
yet. Are you still up for a quick revision?

João Pedro

File: Languages/French/DefInjected/JobDef/Jobs.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<LanguageData>
+
+  <!--EN: learning TargetB.-->
+  <LearnTech.reportString>Apprend TargetB.</LearnTech.reportString>
+  <!--EN: documenting TargetB.-->
+  <DocumentTech.reportString>Documente TargetB.</DocumentTech.reportString>
+  <!--EN: researching TargetB.-->
+  <ResearchTech.reportString>Recherche TargetB.</ResearchTech.reportString>
+  <!--EN: practicing TargetB.-->
+  <TrainWeapon.reportString>S'entraîne avec TargetB.</TrainWeapon.reportString>
```
Comment (jptrrs) [inline Languages/French/DefInjected/JobDef/Jobs.xml]: We lost the "Target B" on these lines to leave just the verb.

**Review by jptrrs — CHANGES_REQUESTED** [review]
Hi there, Rémi!
I reviewed your translations and it seems just fine. I left a note, but the only major thing we would need to add would be the new lines for our next update. I added them to your keys file and explained it there. 
Hope you're still up to the task!

**Comment by Arsaell** [comment]
Hi there !

I just applied the changes you requested, it should be ok for now...
Don't hesitate to come to me if there are other updates you need translated !

Keep it going, and have a nice one !
PN.

**Review by jptrrs — APPROVED** [review]
Great! On parle français maintenant! Merci.

**Comment by jptrrs** [comment]
Hey, Rémi!
I'm writing here because I really have no other means of contacting you. 
Since your translation, we have added a bunch of new lines to the mod. It would be nice if you could update it! The changes were mostly on [the main keys file](https://github.com/jptrrs/HumanResources/blob/master/Languages/French/Keyed/HumanResources_keys.xml) and on the [Jobs file](https://github.com/jptrrs/HumanResources/blob/master/Languages/French/DefInjected/JobDef/Jobs.xml). I've marked them all with `<!--NEW-->`.
Thank you very much!

**Pull request merged**